modular_zubbers/code/modules/knitting/knitting.dm

```diff
@@ -0,0 +1,194 @@
+/obj/item/knittingneedles
+	name = "knitting needles"
+	desc = "Silver knitting needles used for stitching yarn."
+	icon = 'knitting.dmi'
+	inhand_icon_state = "knittingneedles"
+	lefthand_file = "knittingneedles_lh"
+	righthand_file = "knittingneedles_rh"
+	belt_icon_state = "knittingneedles"
+	icon_state = "knittingneedles"
+	slot_flags = ITEM_SLOT_BELT
+	w_class = WEIGHT_CLASS_SMALL
+	throwforce = 5
+	throw_speed = 3
+	throw_range = 5
+	attack_verb_continuous = list("stabs")
+	attack_verb_simple = list("stab")
+	sharpness = SHARP_POINTY
+
+	var/working = FALSE
+	var/obj/item/yarn/ball
+	var/static/list/knitables = list(/obj/item/clothing/head/beret/knitted,
+	/obj/item/clothing/suit/costume/ianshirt,
+	/obj/item/clothing/suit/toggle/jacket/sweater,
+	/obj/item/clothing/suit/costume/ghost_sheet,
+	/obj/item/clothing/neck/scarf/knitted,
+	/obj/item/clothing/head/beanie/knitted,
+	/obj/item/clothing/gloves/color/grey/protects_cold,
+	/obj/item/clothing/suit/hooded/wintercoat/skyrat,
+	/obj/item/clothing/neck/mantle,
+	/obj/item/clothing/accessory/armband/knitted,
+	/obj/item/clothing/under/misc/pj/red,)//Add more? Get Radial thumbnails fixed.
+	var/static/list/name2knit
```

Comment:
A static list does not work here like this. I think instead you should make this a global instead.

Reply:
To clarify, is this referring to name2knit here?

Reply:
Yes, name2knit.